packages/admin-ui/src/calendar/calendar-day-title.ts

```diff
@@ -1,5 +1,5 @@
 import { createComponent, useElement } from '@vtex/admin-ui-react'
-import { Role } from 'reakit/Role'
+import { Role } from 'reakit'
```

Comment:
Importing this way works fine o Reakit the problem is on Ariakit. When we changed the way of importing our bundle size reduced considerably. Ref: https://reakit.io/docs/bundle-size/